src/fileformat/file_format/elf/elf_format.cpp

```diff
@@ -2088,6 +2088,18 @@ void ElfFormat::loadSegments()
 		fSeg->setElfAlign(seg->get_align());
 		fSeg->load(this);
 		segments.push_back(fSeg);
+
```

Comment:
Maybe move this whole block to a separate function? Something like `checkSegment()`, `checkSegmentLoadable()`, etc.